alg/teca_latitude_damper.cxx

```diff
@@ -160,6 +220,36 @@ std::vector<teca_metadata> teca_latitude_damper::get_upstream_request(
 
     arrays.insert(damped_vars.begin(), damped_vars.end());
 
+    // Cleaning off the postfix for arrays passed in the pipeline. 
```

Comment:
looks good. 

this is a common use case. can you throw the code into a utility function? the question is where does it go, maybe `core/teca_metadata_util.{h,cxx}`?